### PR TITLE
Disabling scrolling if dragToDismissDisabled is true in the rendering_complete event

### PR DESCRIPTION
**Pull request by emawby**
This PR is intended to disable drag to dismiss for IAM carousel. This change is agnostic to carousel in case we want to disable scrolling for other reasons in the future

## 
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-android-sdk/1199)

### CODE REVIEW

File: Examples/OneSignalDemo/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -10,13 +9,6 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
     <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
 
-    <permission
```
Comment (jkasten2) [inline Examples/OneSignalDemo/app/src/main/AndroidManifest.xml:13]: amazon changes do not relate to this PR, should revert this change in the PR

**Review by jkasten2 — APPROVED** [review]

File: Examples/OneSignalDemo/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (emawby) [reply Examples/OneSignalDemo/app/src/main/AndroidManifest.xml:13]: Gotcha, I wanted to change the default from using Amazon to not using Amazon, but I can make a separate PR for that
